ts/parts/Time.ts

```diff
@@ -395,16 +405,100 @@ class Time {
 
     public getTimezoneOffset: ReturnType<Time['timezoneOffsetFunction']>;
 
-    public get?: (unit: string, date: Date) => number;
-
-    public set?: (unit: string, date: Date, value: number) => (number|undefined);
-
     /* *
      *
      *  Functions
      *
      * */
 
+    /**
+     * Get the value of a date object in given units, and subject to the Time
+     * object's current timezone settings. This function corresponds directly to
+     * JavaScripts `Date.getXXX / Date.getUTCXXX`, so instead of calling
+     * `date.getHours()` or `date.getUTCHours()` we will call
+     * `time.get('Hours')`.
+     *
+     * @function Highcharts.Time#get
+     *
+     * @param {Highcharts.TimeUnit} unit
```

Comment:
In public (API-docs-sense) members all types have to be available as a doclet. The name should end on `Value` as it contains string literals only. So in this case, the minimum is
```ts
/**
 * Time units
 * @typedef {"Date"|"Day"|"FullYear"|"Hours"|"Milliseconds"|"Minutes"|"Month"|"Seconds"} Highcharts.TimeUnitValue
 */
```

Reply:
Done, thanks.